qutebrowser/utils/urlmatch.py

```diff
@@ -30,6 +30,7 @@
 import urllib.parse
 
 from qutebrowser.utils import utils, qtutils
+from PyQt5.QtCore import QUrl
```

Comment:
nitpick: Please move this to a separate block before the `from qutebrowser` import, see [pep8](https://www.python.org/dev/peps/pep-0008/#imports).

Reply:
Thanks